site/gatsby-site/src/graphql/classifications.js

```diff
@@ -0,0 +1,122 @@
+import gql from 'graphql-tag';
+
+export const FIND_RESOURCE_CLASSIFICATION = gql`
+  query FindResourceClassifications($query: ResourceQueryInput) {
+    resources(query: $query) {
+      _id
+      incident_id
+      notes
+      classifications {
+        DatasheetsForDatasets
+        Publish
+      }
+    }
+  }
+`;
+
+export const UPDATE_RESOURCE_CLASSIFICATION = gql`
+  mutation UpsertResourceClassification($query: ResourceQueryInput, $data: ResourceInsertInput!) {
+    upsertOneResource(query: $query, data: $data) {
+      _id
+      incident_id
+      notes
+      classifications {
+        DatasheetsForDatasets
+        Publish
+      }
+    }
+  }
+`;
+
+export const FIND_CSET_CLASSIFICATION = gql`
+  query FindCSETClassifications($query: ClassificationQueryInput) {
+    classifications(query: $query) {
+      _id
+      incident_id
+      notes
+      classifications {
+        AIApplications
```

Comment:
I have usually seen this with underscores separating the words. Is this the more common pattern for keys with multiple words in graphql?

Reply:
Graphql convention is camel case, but since there are no spaces it leaves it as it is.

Fields with spaces are joined and camel-cased: `Datasheets for Datasets` => `DatasheetsForDatasets`